package/MDAnalysis/core/topology.py

```diff
@@ -165,7 +169,7 @@ class TransTable(object):
 
     Attributes
     ----------
-    n_atoms, n_residues, n_segments : int
+    `n_atoms`, n_residues, n_segments : int
```

Comment:
Even if escaping the first one works, it seems inconsistent with all the other docs that do not have backtics on variable names or argument names in these lists.